test/serializer_test.rb

```diff
@@ -1363,6 +1363,7 @@ def name
   def test_inheritance_does_not_used_cached_attributes
     parent = Class.new(ActiveModel::Serializer) do
       attributes :title
+      alias :read_attribute_for_serialization :send
```

Comment:
This is needed? If so, it probably shouldn't just be in the test, but generally in the serializer, no?

Also, see https://github.com/rails/rails/blob/4-2-stable/activemodel/lib/active_model/serialization.rb#L141